### DISCUSSION TITLE
A bit confused...

### DISCUSSION THREAD

**Discussion by gloparco** [forwardemail/forwardemail.net#216] [General]
I stumbled upon forwardemail when investigating potential solutions for email verifications during registration, forgotten passwords and the like.  I now understand the term "transactional email" and notice that forwardemail strictly forbids other types of emails, namely "marketing email".

My confusion (and it remains even after looking at the FAQ's and googling), is what do I do if I have a single domain for my business and want to be able to send transactional emails, in addition to marketing emails, and just everyday 1-on-1 sending and receiving of emails with customers, potential customers, partners, etc. (for things like answering questions, responding to feedback, etc)?

Since it is a single domain, it's not like I can configure two MX records, one for transactional emails and one for all other emails.

So is this service not for me?  I notice that a lot of competitors offer an all-in-one solution, but it looks like this service is limited to transactional and incoming emails only.

Thanks in advance for clarifying.

**Comment by titanism** [comment]
Hi there @gloparco - we plan to add newsletter support soon (it's next on our roadmap this month after we finish CalDAV/Calendar implementation this week).  You could use a service like Mailchimp in the interim.

**Reply by titanism** [reply]
To clarify, mainly we don't allow mass newsletters right now (e.g. emails with all the same subject to a huge audience).   When we add newsletter support, there will be a UI for creating templates, managing lists and subscribers, sending campaigns, monitoring delivery, and more.  It will also add the `List-Unsubscribe` header to emails for you.

**Reply by gloparco** [reply]
Thanks, but I am still a bit confused.  If my MX record is set up with forwardemail, how would I utilize another service for marketing emails?  Are you saying that I could use their service only if I do not use my custom domain name?  That would mean that the marketing emails to customers would come from mailchimp and not my custom domain, unless there is something I am not understanding.

**Reply by titanism** [reply]
MX is for inbound receiving mail, not for sending mail.  I think you are a bit confused.  You can use multiple email services for outbound mail (e.g. marketing/transactional/etc).  You just need to ensure you have SPF/DKIM/DMARC configured properly.  Again, MX is only for inbound mail (e.g. IMAP storage).

**Reply by gloparco** [reply]
Hmm, yes, thanks for clarifying.  Not sure why I thought MX records were for outgoing.  If anyone else is reading this, the SPF/DKIM/DMARC are defined using TXT DNS entries.  I guess when setting them up in the past, I just never looked into what those TXT entries were for.  I had just assumed it was a way for google or other services to confirm that you own the domain.

Anyway, thanks for setting me straight!  It sounds like I can accomplish what I am looking to do using forwardemail, sans the marketing email.

In terms of how it all fits together, if I use forwardemail, then I would be able to receive mail using the service, and I would utilize the service for all of my outgoing transactional emails.  Then, for marketing emails, or perhaps even the daily emails I referred to, I would utilize another smtp provider, so that my outgoing non-transactional mails do not count toward your monthly limits.  Is that correct?

**Reply by titanism** [reply]
> I would utilize another smtp provider, so that my outgoing non-transactional mails do not count toward your monthly limits. Is that correct?

Correct, other providers usually just ask you to include them in your SPF record and possibly add a TXT record for domain verification (like we do).  We're aiming to have newsletter support out by the end of this month though.

**Reply by gloparco** [reply]
Great, looking forward to that offering.  Any info on what it might cost?

**Reply by titanism** [reply]
Probably included to a certain limit for free (like we've always done to honor our loyal customers)

Then more tiers for different usage, and a paid add-on for dedicated IP addresses

**Comment by gloparco** [comment]
Also, while I'm at it - another question for you: if I have a web app that, say monitors news, and I want to send all users a daily update of news that they subscribed to, would that be considered "marketing email"?  I ask because it certainly is not marketing in the true sense of the word, but it is also not transactional, and it would be 1 email per customer, per day, perhaps even with the same subject line (maybe something like "Your daily news summary".  For forwardemail purposes, would that be considered marketing email?

**Reply by titanism** [reply]
Hi there, this might be OK as long as the email is tailored to them and they opted-in to begin with.  If they report or mark it as spam there might be issues.

**Reply by gloparco** [reply]
Got it, thanks

**Comment by gloparco** [comment]
It's been a few months... any update on the newsletter service?

**Reply by titanism** [reply]
It will be released this year, still planned!
